docs/integrations/vector-databases/overview.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Overview - Vector Databases
+description: A summary of what vector databases are.
+search:
+  boost: 3
+---
+
+## Overview - Vector Databases
```

Comment:
For some reason you've done the headings here as `##` they should be `#` for the top level heading.